src/clj/collect_earth_online/db/doi.clj

```diff
@@ -0,0 +1,91 @@
+(ns collect-earth-online.db.doi
+  (:require [clj-http.client :as http]
+            [clojure.java.io :as io]
+            [triangulum.database :refer [call-sql
+                                         sql-primitive]]
+            [triangulum.config  :refer [get-config]]
+            [triangulum.type-conversion :as tc]
+            [collect-earth-online.generators.external-file :refer [create-shape-files]])
+  (:import java.time.format.DateTimeFormatter
+           java.time.LocalDateTime))
+
+(def base-url (:url (get-config :zenodo)))
+
+(defn req-headers
+  []
+  (let [auth-token (:api-key (get-config :zenodo))]
+    {"Authorization" (str "Bearer " auth-token)}))
+
+(defn get-zenodo-deposition
+  [deposition-id]
+  (let [headers (req-headers)]
+    (http/get (str base-url "/deposit/depositions/" deposition-id) {:headers headers :as :json})))
+
+(defn create-contributors-list
+  [users institution-name]
+  (map (fn [user]
+         {:name        (:email user)
+          :type        "DataCurator"
+          :affiliation institution-name})
+       users))
+
+(defn create-zenodo-deposition
+  [institution-name
+   project-name
+   user users-assigned
```

Comment:
Format the arguments (user, user-assigned) by placing them on separate lines, similar to the other